lib/rust/ensogl/component/list-view/src/lib.rs

```diff
@@ -467,6 +467,8 @@ impl<E: Entry> application::command::FrpNetworkProvider for ListView<E> {
 }
 
 impl<E: Entry> application::View for ListView<E> {
+    // NOTE: if implementing a different specialization of ListView, make sure to change the label
+    // appropriately so that 'register' would properly differentiate shortcuts etc.
```

Comment:
I'm not sure if this comment is right. First, I'm not sure what is a different specialization - Rust does not have specializations in c++ style. And the entry does not affect the way the shortcut are handled, so it fact each ListView should have the same label.

Reply:
Indeed. Now that I think of it, if guidelines should be provided anywhere, I'd rather expect to see them in View trait's docs, forming a part of the trait's contract description. With that said, I'm more than happy to remove it from here. It's not however clear to me what are the rules around the various methods of View, so I don't currently feel confident to be adding docs to View.

Do you think it'd be ok for me to just remove this comment here, or would you suggest I try adding comments in View, or somewhere else?